src/derivers/bip39.ts

```diff
@@ -30,7 +31,10 @@ export async function deriveChildKey({
   path,
   curve,
 }: DeriveChildKeyArgs): Promise<SLIP10Node> {
-  return createBip39KeyFromSeed(await mnemonicToSeed(path), curve);
+  return createBip39KeyFromSeed(
+    mnemonicToSeedSync(path, englishWordlist),
```

Comment:
Hmm, why the switch to the synchronous version of this function?

Reply:
We [did not update the async version](https://github.com/MetaMask/scure-bip39/blob/main/src/index.ts#L137) of the function, I don't remember why, but I assumed we wanted to start using the sync version given this fact.

Reply:
Perhaps we should update the async version too?

Reply:
Ah, good catch.

This might be a genuine performance concern. The async implementation will use `nextTick` to split the work up to avoid blocking the main thread. Key generation can be slow on some machines.

I guess this works for now, but it might be worth our time to update the async version of this function as well.